templates/css_declarations.erb

```diff
@@ -0,0 +1,3 @@
+<% @css_sheets.each do |sheet| -%>
+  <link href="<%= CGI::escapeHTML( sheet[ 'local' ] ? sheet[ 'filename' ] : 'proxy.php?url=' + CGI::escape( sheet[ 'filename' ] ) ) %>" rel="stylesheet" type="text/css" />
```

Comment:
Did you test this? It's strange you didn't have to add `require 'cgi'` to the template.